var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -82,7 +82,10 @@ def get_rocm_prefix_info(self):
                           for name in ('llvm-amdgpu', 'hsa-rocr-dev',
                                        'rocminfo', 'rocm-device-libs'))
             mydict['rocm-path'] = self.spec.prefix
-            device_lib_path = mydict['rocm-device-libs'].amdgcn.bitcode
+            if '@:3.8.0' in self.spec: 
```

Comment:
Should this be `self.spec.satisfies('@:3.8.0'):`?

Reply:
To be fair, the whole file is using `[version] in [spec]`, so for consistency this is better.

Reply:
> To be fair, the whole file is using `[version] in [spec]`, so for consistency this is better.

I should've looked at more than the change then.